wgengine/monitor/monitor_test.go

```diff
@@ -28,3 +31,35 @@ func TestMonitorJustClose(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+var monitor = flag.String("monitor", "", `go into monitor mode like 'route monitor'; test never terminates. Value can be either "raw" or "callback"`)
```

Comment:
TIL you can bolt flags onto tests. Thanks!

Reply:
🔩